homeassistant/components/aladdin_connect/cover.py

```diff
@@ -72,6 +72,7 @@ def __init__(self, acc: AladdinConnectClient, device: DoorDevice) -> None:
         self._number = device["door_number"]
         self._attr_name = device["name"]
         self._attr_unique_id = f"{self._device_id}-{self._number}"
+        self._attr_is_closed = None
```

Comment:
I think this change isn't correct in this case, I think `add_entities` in `setup_platform` needs to hint for updating this entity before adding it

Reply:
I agree this is the better way to do it. I was not aware of this prior. Looking at the docs, I have added `update_before_add` to the `add_entity` call in `setup_platform`.